qt/python/mantidqt/mantidqt/widgets/codeeditor/multifileinterpreter.py

```diff
@@ -403,3 +443,32 @@ def zoom_out_all_tabs(self):
     def on_completion_change(self):
         for idx in range(self.editor_count):
             self.editor_at(idx).setCompletion(self.completion_enabled)
+
+    def focus_in(self, file):
+        self.check_file_for_external_modifications(file)
+
```

Comment:
do we need this function if it just calls another with the same arguments?

Reply:
I could inline the function, but I thought it would be more descriptive to use a separate function, plus maybe someone will want to do something else inside focus_in(). I don't mind, could inline it if it's clearer.